magento/pwa-studio
# PB-363: Update the Page Builder integration docs

## Description

This PR updates the Page Builder integration docs to indicate that the integration is only compatible with Magento Commerce 2.3.4+. I also adds a new diagram, updates existing diagrams, and clarifies existing content and descriptions for the integration overview page.

## Related Issue

https://jira.corp.magento.com/browse/PB-363

## Acceptance 

@davemacaulay 
@jcalcaben 
@tkacheva 

### Verification Steps

1. Navigate to preview instance here: https://bdenham.github.io/pb-to-pwa-docs/pagebuilder/.
2. Ensure that the information in the callout at the top of the page satisfies the information about compatibility with Magento Commerce 2.3.4+.
3. Ensure that remaining information on this page is correct.
